src/App.js

```diff
@@ -1,24 +1,40 @@
-import Footer from './components/Footer';
-import Main from './components/Main';
-import NavBar from './components/NavBar';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App"> 
+import Footer from "./components/Footer";
+import Main from "./components/Main";
+import NavBar from "./components/NavBar";
+import React, { useState, useEffect } from "react";
+
```

Comment:
Please remove empty line from here too.

Reply:
I have made the requested changes, @NegiAkash890 kindly check now.